quickstarts/blazor/config.yml

```
@@ -0,0 +1,48 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: blazor
+
+# Displayed in the UI (required)
+title: Blazor
+name: Blazor
+
+# Long-form description of the quickstart (required)
+description: |
+  The Blazor quickstart allows you to get visibility into the performance of your Blazor application.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Get an actionable look at your Blazor applications performance in seconds with the Blazor quickstart for New Relic.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Jack de la Motte
+  - Cesar Pichardo
+  - Michelle Yi
+  - Dabel Obikili
+  - Bria Brewton
+  - Yasir Basit
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - Blazor
+  - .NET
+  - .NET Core
+  - C#
+
+alerts:
+  - High CPU Usage
+  - High Error Rate
+dashboards:
+  - Blazor
```

This needs to match the directory with the dashboard, so for this one it would just be `blazor` (no uppercase "B")
```suggestion
dashboards:
  - blazor
```